packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -255,10 +255,11 @@ function usePositions(
                       calloutElement.current,
                       elementPositions,
                     );
-
                 setCalloutPositions(newElementPositions);
               }
             });
+
+            return () => cleanupObserver();
```

Comment:
Does this work? I don't think `requestAnimationFrame` supports returning a cleanup function. 

I'd think you'd need to do the cleanup in the function returned from `useEffect` on line 266.